tests/util.ts

```diff
@@ -0,0 +1,22 @@
+import {ChildProcess, spawn} from 'child_process';
```

Comment:
Please add spaces to this imports
```suggestion
import { ChildProcess, spawn } from 'child_process';
```